src/main/java/uk/gov/hmcts/reform/unspec/model/BusinessProcess.java

```diff
@@ -1,14 +1,46 @@
 package uk.gov.hmcts.reform.unspec.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.Builder;
 import lombok.Data;
 import uk.gov.hmcts.reform.unspec.enums.BusinessProcessStatus;
 
+import static java.util.Optional.ofNullable;
+
 @Data
 @Builder
 public class BusinessProcess {
 
-    private final String processInstanceId;
-    private final BusinessProcessStatus status;
-    private final String activityId;
+    private String processInstanceId;
+    private BusinessProcessStatus status;
+    private String activityId;
+
+    @JsonIgnore
+    public boolean hasSimilarProcessInstanceId(String processInstanceId) {
+        return this.getProcessInstanceId().equals(processInstanceId);
+    }
+
+    @JsonIgnore
+    public BusinessProcessStatus getStatusOrDefault() {
+        return ofNullable(this.getStatus()).orElse(BusinessProcessStatus.READY);
+    }
+
+    @JsonIgnore
+    public BusinessProcess start() {
+        this.status = BusinessProcessStatus.STARTED;
+        this.activityId = null;
+        return this;
+    }
+
+    @JsonIgnore
+    public BusinessProcess updateProcessInstanceId(String processInstanceId) {
+        this.processInstanceId = processInstanceId;
+        return this;
+    }
+
+    @JsonIgnore
+    public BusinessProcess updateActivityId(String activityId) {
+        this.activityId = activityId;
+        return this;
+    }
```

Comment:
Do we really need these methods? They're all used only once, we could replace them with `toBuilder()` calls on business process that needs updating.